content/projects/tdd/simple-calculator-part1/_index.md

```diff
@@ -160,4 +160,16 @@ multiply(1,3)
 // should still return 3
 multiply(-1,3)
 // should still return -3
-```
\ No newline at end of file
+```
+
+## Instructions for reviewer
+
+* File structure is important, expected file structure displayed above.
+* Code should be DRY code and clean code.
```

Comment:
Review instructions should be project specific, please remove the line about clean and dry code as all projects submissions should be clean and dry etc